### DISCUSSION TITLE
Default Prompt Looks Wrong

### DISCUSSION THREAD

**Discussion by brianpham** [starship/starship#2306] [Q&A] [answered]
Hi,

I recently install starship and wondering why my prompt looks like this. My configuration file is empty.
But it shows this. I am not too sure where it is coming from. Is it normal? From the docs, it looks like the default prompt should be just a dir name.

```
~ on ☁️  (us-east-1) on ☁️ bpham@google.com(us-west1)
❯
```

**Comment by davidkna** [comment, marked as answer]
This looks like one of the cloud modules like [aws](https://starship.rs/config/#aws) or [gcloud](https://starship.rs/config/#gcloud). These generally get enabled by certain environment variables being set and you can disable them in your config with:
```toml
# ~/.config/starship.toml
[aws]
disabled = true

[gcloud]
disabled = true
```

To find out which module is responsible for which segment in your prompt, you can run `starship explain`.

**Reply by brianpham** [reply]
Ahh thank you for the quick response. Was able to disable those and the `starship explain` command helped a lot.
